### DISCUSSION TITLE
Is it possible to load a folder instead of a static file for RAG?

### DISCUSSION THREAD

**Discussion by zachrattner** [open-webui/open-webui#1250] [Q&A]
I want to use OpenWebUI to ask questions about a source code repo. Is it possible to link a folder to the Documents tab instead of an individual file?

**Comment by pwab** [comment]
Not the same request but related to:

- https://github.com/open-webui/open-webui/discussions/995.

- https://github.com/open-webui/open-webui/issues/529
